ADA/ada.qmd

```diff
@@ -19,57 +17,194 @@ It brings together:
+
+---
+
+## What this means for existing code
+
+### SQL scripts
+
+---
+
+#### Scripts currently used in SSMS
+
+If you currently use data and scripts stored in SSMS for your statistics publication pipeline, there will be small changes to your production pipelines that vary depending on your current process. 
+
+Your SQL code will need to be migrated over to Databricks to allow it to read from the Databricks SQL Warehouse and will require some minor changes to syntax as SSMS uses T-SQL and Databricks uses Spark SQL. The ADA team are currently developing a tool to allow you to easily convert T-SQL queries to Spark SQL syntax. 
+
+You can write new SQL code in the Databricks SQL Editor, which is very similar to SSMS. 
+
+---
+
+#### SQL scripts in repositories or embedded in R scripts
+
+These scripts do not need to be moved, but the syntax of the scripts or embedded code will need to be rewritten from T-SQL to Spark SQL. The only thing you will need to change is the connection source to allow the code to run. Rather than pointing to data held in current SQL databases, you will need to point towards the Databricks Delta Lake instead. 
+
+
+---
+
+#### Workflows
+
+---
+
+If you'd like to make use of automated workflows to run scheduled code, then your SQL code will need to be moved into Databricks notebooks, using Spark SQL syntax. 
+
+---
+
+### R scripts in RStudio
+
+---
+
+::: {.callout-important}
+If you have an existing RAP process and pipeline set up using RStudio and SSMS, there is no current expectation for you to migrate your existing R code or scripts from RStudio to Databricks notebooks. 
+:::
+
+Your R code can remain in RStudio and all existing R processes can still be run there. Code that points towards SQL scripts will need to be redirected to Databricks rather than SSMS. In order to use RStudio with a Databricks SQL warehouse, you will need to manually set up a connection. You can find more information about that on our [setup guidance page](/ADA/databricks_rstudio_sql_warehouse.html). 
 
 ---
 
-## Databricks code templates
+### Using notebooks
 
-See examples and code snippets in R, SQL and Python in these [Databricks code template notebooks](https://adb-6882499576863257.17.azuredatabricks.net/browse/folders/3070168888967919?o=6882499576863257#)
+---
+
+If you prefer using R in a notebook format and wish to use notebooks for your analysis, this is fine, but please consider the skill sets and knowledge levels in your team when doing so. For the purposes of RAP, analysis should be reproducible. If the rest of your team are unfamiliar with notebooks and will need to pick up your work at a later date, this could cause issues. The [Statistics Development Team](mailto:statistics.development@education.gov.uk) are happy to advise on this and provide guidance if you have any concerns. 
```

Comment:
Worth adding something to highlight databricks brings the functionality of workbooks? I think it'll be new to most people unless they're python users.

Reply:
Do you mean workspaces or notebooks? I think notebooks but just want to make sure

Reply:
I did mean notebooks! (sorry) 
I know its referenced in detail already but I imagine some people will just be like why does everyone care about notebooks all of a sudden (thinking leadership people)

Reply:
Think we've covered this as part of the paragraph added in Cam's comment below